modal.js

```diff
@@ -0,0 +1,213 @@
+  
+      <p>
+      ${items.description}
+      </p>
+      <div class="language1-inline">
+        <ul>
+ 
+        </ul>
+      </div>
+      <button type="button" class="btn-one1">See project</button>
+    </div>
+  </div>
+  </div>`;
+  });
+}
+createCardsMobile();
+
+const seeMe1 = document.querySelectorAll('.btn-one1');
+const seeMe2 = document.querySelectorAll('.btn-two');
+
+const desktopPopupText = " Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been  industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea";
+
+const popupStringLiteral = `
+  <div class="popup-container">
+      <h1>Tonic</h1>
+      <img
+        src="./images/popup/close-icon.svg"
+        alt="close image"
+        class="close-img"
+      />
+      <img src="./images/popup/frame.svg" alt="canopy" class="popup-cano" />
+
+      <img src="./images/popup/photo1.png" alt="#" class="popup-photo display-only-mobile" />
+      <img src="./images/popup/popup-desktop.png" alt="#" class="popup-photo display-only-desktop" />
+
+      <div class="text-and-tags">
+        <p class="text display-only-mobile" >
+          Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. It has survived not only
+          five centuries, but also the leap into electronic typesetting,
+          remaining essent
+        </p>
+
+        <p class="display-only-desktop">${desktopPopupText} </p>
+
+        <div class="tags-live-source-envelope">
+          <img
+            src="./images/popup/mobile-tags.svg"
+            alt="mobile tags"
+            class="tags1"
+          />
+          <img
+            src="./images/popup/tags-desktop.svg"
+            alt="tags desktop"
+            class="display-on-desktop"
+          />
+
+          <div class="source-live">
+            <img src="./images/popup/see-live.png" alt="see live" />
+            <img src="./images/popup/see-source.png" alt="see source" />
+          </div>
+        </div>
+      </div>
+    </div>
+`;
```

Comment:
- [ ] I noticed that the popup window is fixed and similar across all cards.  It should be created dynamically using the information stored in that JavaScript object `portfolioDetails`. so kindly, make the modal popup created with the data from `portfolioDetails` array ( the same way you implemented the cards dynamically )👌💯

![image](https://user-images.githubusercontent.com/45231276/180896578-ff5504bf-09af-43ff-8d18-cb23a0d887dd.png)